extensions/default/src/commandsModule.js

```diff
@@ -2,9 +2,17 @@ const commandsModule = ({ servicesManager, commandsManager }) => {
   const {
     MeasurementService,
     HangingProtocolService,
+    UINotificationService,
   } = servicesManager.services;
 
   const actions = {
+    displayNotification: ({ text }) => {
+      UINotificationService.show({
+        title: 'RectangleROI Threshold Tip',
```

Comment:
Why is this in the default extension? This is specifically for the RectangleROI. An OHIF mode using default might not even be using Cornerstone.